stable/minecraft/templates/NOTES.txt

```diff
@@ -1,10 +1,18 @@
 {{- if eq (printf "%s" .Values.minecraftServer.eula) "FALSE" }}
 ##############################################################################
-####   ERROR: You did not agree to the EULA in your 'helm install' call   ####
-####     helm delete {{ .Release.Namespace }} and re-run install with     ####
-####       --set minecraftServer.eula="true" added (see README.md).       ####
+####   ERROR: You did not agree to the EULA in your 'helm install' call.  ####
 ##############################################################################
-{{- end }}
+
+This deployment will be incomplete and need to be deleted:
+
+    helm delete {{ .Release.Name }}
```

Comment:
LGTM as is, but a quicker way, is to keep the release and add the deployment by performing an upgrade:
```
helm upgrade {{ .Release.Name }} \
  --set minecraftServer.eula=true stable/minecraft
```
works nicely.

Up to you, one potential problem could be that I'm not 100% sure that upgrading a release to the same chart version is actually supported. cc/ @michelleN